miner/worker_builder.go

```diff
@@ -45,24 +45,22 @@ func (w *worker) fillTransactionsAndBundles(interruptCh chan int32, env *environ
 	{
 		bundles := w.eth.TxPool().PendingBundles(env.header.Number.Uint64(), env.header.Time)
 
-		// if no bundles, not necessary to fill transactions
-		if len(bundles) == 0 {
-			return errors.New("no bundles in bundle pool")
-		}
+		// if no bundles, skip bundle commit
```

Comment:
Why line 38 `env.header.GasLimit /= 2`? Does it mean that only 50% gas can be used for bundles?

Reply:
true